swift-sdk/IterableInboxCell.swift

```diff
@@ -38,4 +38,15 @@ open class IterableInboxCell: UITableViewCell {
         // Initialization code
     }
 
+    // override this to show unreadCircle color when highlighted
+    // otherwise the background color is not correct.
+    open override func setHighlighted(_ highlighted: Bool, animated: Bool) {
+        let color = unreadCircleView?.backgroundColor
```

Comment:
Can you move this def of color inside of the if highlighted block?

Reply:
We can't add this to the `if highlighted` block because otherwise the original color will be overwritten. We have to save the original color before calling the super method.

Reply:
Got it